Marlin/example_configurations/RepRapWorld/Megatronics/Configuration.h

```diff
@@ -1123,6 +1123,15 @@ const bool Z_MIN_PROBE_ENDSTOP_INVERTING = false; // set to true to invert the l
 //
 //#define SAV_3DLCD
 
+// CONTINUE_BUTTON_FEATURE defines a button for user interaction in case you have no display or panel.
+// To continue in a M0/M1, ...
+// When a BEEPER_PIN is defined beeps for one second.
+// Lights a LED while waiting if LED_PIN is defined.
+//#define CONTINUE_BUTTON_FEATURE
+//#define CONTINUE_PIN 35
+//#define BEEPER_PIN 37
+//#define LED_PIN 13
```

Comment:
I don't think these overrides will work, currently. They might, but they will generate complaints from the compiler about multiple defines.